.github/workflows/common-post-jobs.yaml

```diff
@@ -64,3 +64,17 @@ jobs:
           sha: ${{ inputs.sha }}
           status: ${{ github.event_name != 'schedule' && 'success' || 'failure' }}
           description: 'Skipped'
+
+  fail-workflow:
```

Comment:
This comment refers to `commit-status-final` above.

The following hunk could be now deleted, as no longer applicable:

```yaml
- name: Set final commit status
  if: ${{ inputs.result == 'skipped' }}
  uses: myrotvorets/set-commit-status-action@3730c0a348a2ace3c110851bed53331bc6406e9f # v2.0.1
  with:
    sha: ${{ inputs.sha }}
    status: ${{ github.event_name != 'schedule' && 'success' || 'failure' }}
    description: 'Skipped'
```

And similarly the `if: ${{ inputs.result != 'skipped' }}` check above.